rotkehlchen/globaldb/schema.py

```diff
@@ -238,6 +238,16 @@
 );
 """
 
+DB_CREATE_GENERAL_CACHE = """
+CREATE TABLE IF NOT EXISTS general_cache (
+    type CHAR(1) NOT NULL,
+    name TEXT NOT NULL,
+    value TEXT,
+    last_queried_ts INTEGER,
```

Comment:
Why is the `last_queried_ts` allowed to be NULL? If there is a cache entry it has to come with a timestamp.